exercises/practice/robot-simulator/.meta/config.json

```diff
@@ -12,7 +12,7 @@
     "uzilan"
   ],
   "files": {
-    "solution": ["src/main/kotlin/RobotSimulator.kt"],
+    "solution": ["src/main/kotlin/Robot.kt"],
```

Comment:
Should this be edited by hand or should it be altered via configlet?

Reply:
The `config.json` needs to be edited by hand :)